DSharpPlus/Net/Abstractions/Transport/TransportUser.cs

```diff
@@ -33,8 +33,11 @@ internal class TransportUser
         [JsonProperty("username", NullValueHandling = NullValueHandling.Ignore)]
         public string Username { get; internal set; }
 
+        [JsonProperty("global_name", NullValueHandling = NullValueHandling.Ignore)]
+        public string GlobalDisplayName { get; internal set; }
```

Comment:
why is this not called GlobalName?

Reply:
Good question.

Reply:
I actually had the same question when I dug into the source lol